tier2/steam_api.cpp

```diff
@@ -100,6 +100,9 @@ void LoadSteamClient017()
 	}
 
 	steamClientModule = Sys_LoadModule(libpath.c_str());
+#elseif OSX
+	// TODO: This is probably why it cant find the surface we want.
+	steamClientModule = Sys_LoadModule("");
```

Comment:
Try `Sys_LoadModule("steamclient.dylib")` (Steam client library name may be different on macOS). If it doesn't work, you'll need to somehow read list of loaded modules (/proc/self/map on Linux) and use it to locate steam client library. You can also hide the HTML surface warning :)

Reply:
Got it to resolve the correct path to steamclient.dylib. 👍